src/Raven.Client/Documents/Operations/Revisions/RevisionsCollectionConfiguration.cs

```diff
@@ -13,12 +13,15 @@ public class RevisionsCollectionConfiguration
 
         public bool PurgeOnDelete { get; set; }
 
+        public bool IncludeCountersSnapshot { get; set; }
```

Comment:
Under what scenario I _don't_ want that?
We don't give a choice on attachments, and we shouldn't here.

Reply:
removed